src/CMakeLists.txt

```diff
@@ -13,3 +13,19 @@ if(BUILD_TESTING)
    add_subdirectory(test)
    add_subdirectory(examples/cpp)
 endif()
+
+option(BUILD_SITE "Build log4cxx website" OFF)
```

Comment:
Why is this placed here instead of the upper level `CMakeLists.txt` containing `APACHE_MAINTAINER` already? Seems to make more sense to me to have both near by each other.

Reply:
Some of the options are spread over different CMake files.  This seemed like the most logical place to put it at the time, but perhaps not.  We could:
1. Put at top-level CMakeLists
2. Keep where it is
3. Create a new CMakeLists in src/site that contains this logic

Any of these options make sense to me, though perhaps option 3 makes the most sense in terms of how the CMakeLists is currently structured(one CMakeLists per directory).

Reply:
Agreed, your option 3 seems to make most sense.

Reply:
Fixed with the latest commit.